gui/tabs/self_tab.cpp

```diff
@@ -67,7 +67,10 @@ namespace SelfTab {
             {
                 State.Save();
             }
-
+            ImGui::SameLine();
```

Comment:
why do you add the feature on the same line ? it would be better if you increase the window size a bit and make this in it's own line